editor/src/clj/editor/gl/shader.clj

```diff
@@ -340,13 +350,25 @@ This must be submitted to the driver for compilation before you can use it. See
   [^GL2 gl progn loc val]
   ;; No-Op. This is for sampler uniforms. They are just a name. Contains no
   ;; value to set.
-  )
+  nil)
 
 (defmethod set-uniforms-at-index (class (float-array []))
   [^GL2 gl progn loc count vals]
   (let [fb (FloatBuffer/wrap vals)]
     (.glUniform4fv gl loc count fb)))
 
+(defn- shader-type? [value]
+  (case value
+    (:shader-type-vertex :shader-type-fragment #_ :shader-type-compute) true
+    false))
+
+(defn- shader-type->gl-shader-type
+  ^long [shader-type]
+  (case shader-type
+    :shader-type-vertex GL2/GL_VERTEX_SHADER
+    :shader-type-fragment GL2/GL_FRAGMENT_SHADER
+    #_#_ :shader-type-compute GL3/GL_COMPUTE_SHADER))
```

Comment:
What's the significance of this comment?

Reply:
I've included the compute shader constants from the Protobuf enum, but compute shaders are not really supported by the OpenGL profile we currently use in the editor. I'll add a comment to clarify!